javascript/chronometer.js

```diff
@@ -1,26 +1,35 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
   }
-  startClick(callback) {
-    // ... your code goes here
+
+  startClick(){
+    this.intervalId= setInterval(() => { 
+      this.currentTime ++;
+    },1000);
   }
+
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.currentTime / 60);
   }
   getSeconds() {
-    // ... your code goes here
+    return this.currentTime % 60;
   }
   twoDigitsNumber() {
-    // ... your code goes here
+    return this.currentTime.toString( ).padStart(2, '0');
   }
```

Comment:
It would be better if you made your function twoDigitsNumber to accept arguments, that way it will work for a number of use cases.

Also to make sure that we add a 0 when the value is between 0 and 10, we can write a condition
```
twoDigitsNumber(value) {
    if (value < 10) return `0${value}`;
    return `${value}`;
  }